beaker-vue/src/lib/UICell.vue

```diff
@@ -1,11 +1,15 @@
 <template>
   <div
     class="beaker-cell"
+    :class="{
+        selected: props.selected
+    }"
     tabindex="0"
     @keyup.enter.exact="focusEditor"
     @keydown.ctrl.enter.prevent="execute"
     @keydown.shift.enter.prevent="executeAndMove"
     @keyup.esc="unfocusEditor"
+    @click="props.selectCell(props.index || 0)"
```

Comment:
I want to move select up to parent (and remove the need for `props.index`). Reminder to try again.